qatrack/service_log/forms.py

```
@@ -764,8 +770,9 @@ def strigify_form_item(self, item):
                 new = new.str_verbose()
             if old is not None:
                 old = old.str_verbose()
-        elif isinstance(old, int) and issubclass(type(self.fields.get(item)), forms.ModelChoiceField):
-            old = self.fields[item].queryset.model.objects.get(pk=old)
+        elif (isinstance(old, int) or isinstance(old, ModelChoiceIteratorValue)) and issubclass(type(self.fields.get(item)), forms.ModelChoiceField):
```

Ruff (0.15.14)

[warning] Multiple `isinstance` calls for `old`, merge into a single call

Merge `isinstance` calls for `old`

(SIM101)